language/compiler/bytecode-source-map/src/source_map.rs

```diff
@@ -21,7 +21,6 @@ use vm::{
 // Source location mapping
 //***************************************************************************
 
-pub type SourceMap<Location> = Vec<ModuleSourceMap<Location>>;
```

Comment:
Maybe leave this and rename it to `SourceMaps`?

Reply:
The only place it is used is the remap_locations_source_map function, which AFAICS is never called. I'm certainly not opposed to having "SourceMaps" but it doesn't seem justified for one potentially unused function.